addons/udes_stock/tests/common.py

```diff
@@ -291,7 +288,7 @@ def create_picking(cls, picking_type, products_info=False,
 
         if products_info:
             for product_info in products_info:
-                product_info.update(picking=picking)
+                product_info.update(picking=picking, origin=picking.origin)
```

Comment:
Not sure if you need to pass separately origin as a parameter, you are passing the whole picking. I think on create move you could use "origin": picking.origin rather that "origin": origin if that was the reason of passing a new origin parameter

Reply:
Good point, I made changes to follow your suggestion.